src/views/Layout.jsx

```diff
@@ -1,15 +1,7 @@
-import { Outlet } from 'react-router-dom';
+import { Outlet, NavLink } from 'react-router-dom';
 
 import './Layout.css';
 
-/**
- * TODO: The links defined in this file don't work!
- *
- * Instead of anchor element, they should use a component
- * from `react-router-dom` to navigate to the routes
- * defined in `App.jsx`.
- */
-
 export function Layout() {
```

Comment:
Maybe add a short comment about what this component does. As you build out more components and add features, adding simple documentation will help you and others (especially in the future!) have a clear idea of why this component exists. I realize it seems simple for a basic component like this, but it's a good habit to get into moving forward!

Example: 
`/** Layout component that display navigation links in the UI */`